package.json

```diff
@@ -61,7 +61,7 @@
     "@endpass/connect": "^0.32.0-beta",
     "@endpass/faucet": "^0.2.1",
```

Comment:
![image](https://user-images.githubusercontent.com/13414205/83489341-ecad3980-a4b6-11ea-9b03-1c2ce7afeb5d.png)

The same situation with the passport